include/tvm/ir/expr.h

```diff
@@ -107,11 +107,12 @@ class PrimExpr : public BaseExpr {
    * \param value The value to be constructed.
    */
   TVM_DLL PrimExpr(float value);  // NOLINT(*)
+
   /*!
-   * \brief construct from string.
-   * \param str The value to be constructed.
+   * \brief construct from runtime String.
+   * \param value The value to be constructed.
    */
-  TVM_DLL PrimExpr(std::string str);  // NOLINT(*)
+  TVM_DLL PrimExpr(runtime::String value);  // NOLINT(*)
```

Comment:
Can we simply remove this constructor and explicily construct StringImm in the places that we need them

Reply:
Yeah, I consider that as well, but it seems there are too many places using PrimExpr. Need sometime to see which one may take string.

Reply:
Considering the number of files changed so file, I'd like to keep this for now. It seems a bit more work. I can send a separate PR to handle it.